src/pages/valvonta-kaytto/valvonnat.svelte

```diff
@@ -181,6 +184,12 @@
     R.map(Maybe.get),
     R.filter(Maybe.isSome)
   )(query);
+
+  const getTemplateName = (templates, toimenpide) =>
```

Comment:
Jäin nyt vähän miettimään tämän funktion nimeä, kun tämähän palauttaa sen funktion joka muodostaa templaten nimen. Eli joku muu nimi olisi parempi ehkä?

Reply:
Minun puolesta näinkin on ihan hyvä, mutta tuohon voisi olla hyötyä josta kommentista missä on avattu sitä tyyppiä, tyyliin

```
// ((templates: Map<(toimenpideType: number), (templates: Template array)>), toimenpide) -> (template_id: number) -> string
```

Tämä ei ehkä ole ihan formaalin notaation mukainen, mutta kertoisi kuitenkin että tämä on tuommoinen ketjutettu funktio

Reply:
Ja tuota tyyppiä miettiessä (en ole varma sainko ihan oikein sitä) jäi ehkä vielä mielen päälle, että tämä saattaisi mennä yksinkertaisemmaksi, jos tämä ottaisi ihan vain yksiulotteisen arrayn missä on kaikki mahdolliset templatet.